contexts/v1/index.json

```diff
@@ -108,8 +108,8 @@
                 "type": "@type"
             }
         },
-        "AnchorCredentialReference": {
-            "@id": "https://w3id.org/activityanchors#AnchorCredentialReference",
+        "AnchorReference": {
```

Comment:
What's the best vocabulary term here? Ideas:
- `Anchor`
- `AnchorLink`
- `AnchorLocation`
- `AnchorReference`

Reply:
AnchorReference works for me.

Reply:
Note: We can't use simply `Link` as attachment is not part of that data model.